.install.sh

```diff
@@ -21,14 +19,29 @@ pushd h3c
 git pull origin master --tags
 git checkout "$VERSION"
 
-cmake -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON .
-make
-ls -l lib/libh3*
-cp lib/libh3* ../h3/out
-if [ -e ../build ] && [ -d ../build ]; then
-    LIBNAME=`ls ../build/ | grep '^lib'`
-    mkdir -p ../build/$LIBNAME/h3/out
-    cp lib/libh3* ../build/$LIBNAME/h3/out
+if command -v make; then
+  command -v cc  >/dev/null 2>&1 || { echo "cc required but not found."; exit 1; }
+
+  cmake -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON .
+
+  make
+  ls -l lib/libh3*
+  cp lib/libh3* ../h3/out
+  if [ -e ../build ] && [ -d ../build ]; then
+      LIBNAME=`ls ../build/ | grep '^lib'`
+      mkdir -p ../build/$LIBNAME/h3/out
+      cp lib/libh3* ../build/$LIBNAME/h3/out
+  fi
+else
+  # Assumed to be Windows, default to x64
```

Comment:
I'm bit worried maybe some users have installed make in their windows environment? Maybe we can use uname to be more safe here?

Reply:
I'm not sure what `uname` to look for. Maybe anything starting with `MINGW`? On my system it includes the Windows version as part of the OS name.

Another option might be to change from specifying the generator to specifying the architecture with `-A`, in that case we might be able to use CMake to invoke the build system, regardless of which build system is used.

Reply:
@karentycoon Updated, it now uses the `OS` environment variable to detect Windows.